src/api/firebase/firebase.types.ts

```diff
@@ -0,0 +1,13 @@
+/* eslint-disable max-len */
+import firebase from 'api/firebase/firebase.api';
+
+export type Firestore = {
+  Doc: firebase.firestore.QueryDocumentSnapshot<firebase.firestore.DocumentData>;
+};
+
+// Interface for sample documents inside sample collection called "meals"
+export interface Meals {
+  id: string;
+  name: string;
+  ingredients: [{ ingredient: string; portion: string }];
```

Comment:
Please prepare proper interface also for `Ingredient`.

Reply:
What do you mean? 
This interface will probably change in the future as this is just a sample Firestore collection, but if you need change atm please provide more details.

Reply:
Ok, it can stay that way then.